apply-refact.cabal

```diff
@@ -33,7 +33,7 @@ library
   build-depends: base >=4.8 && <4.10
                , refact >= 0.2
                , ghc-exactprint >= 0.5.2
-               , ghc == 8.0.1
+               , ghc >= 8.0.1 && < 8.2
```

Comment:
I think this is very likely to be broken for `HEAD`. Maybe add a 8.1 bound instead of 8.2?